src/main/java/Controller/Handler/RedirectHandler.java

```diff
@@ -0,0 +1,13 @@
+package Controller.Handler;
+import Request.Request;
+import Response.Response;
+
+public class RedirectHandler extends Handler {
+  public Response handle(Request request) {
+
+    return new Response.Builder()
+      .setStatusCode(302)
+      .setHeader("Location", "http://127.0.0.1:3000/")
```

Comment:
You can make this `/` instead of `http://127.0.0.1:3000/`, and then this redirect will work everywhere (instead of just on `127.0.0.1:3000`